### PR TITLE
use const and var when possible

### PR DESCRIPTION
**Pull request by mrturkmen06**
Hi, 

I wished to contribute to project, as first step I would like to make simple changes as summarized below for initial interaction with the project. I hope, I can keep continue to contribute more. 

Summary; 

- `const` and `var` used when possible 
-  If statement changed with switch statement [here](https://github.com/mrturkmen06/passwall-server/blob/ddbde6c8c621ca195cf14d914a9efb58fb6a2391/internal/storage/database.go#L38)
- some variables renamed `(logins > loginList)` 

I thought it could be useful to have some messages and error statements as `const` (string types)  or `var` (errors which returns error type)

### CODE REVIEW

File: internal/storage/bank_account_repository.go [hunk 1]
```diff
@@ -19,14 +19,14 @@ func NewBankAccountRepository(db *gorm.DB) BankAccountRepository {
 
 // All ...
 func (p *BankAccountRepository) All() ([]model.BankAccount, error) {
-	bankAccounts := []model.BankAccount{}
+	var bankAccounts []model.BankAccount
```
Comment (yakuter) [inline internal/storage/bank_account_repository.go:22]: If you do this change, when there is no record on database, the response json object will be null, instead if empty array which is not a good usecase on frontend. Can you revert this change

File: internal/storage/bank_account_repository.go [hunk 2]
```diff
@@ -19,14 +19,14 @@ func NewBankAccountRepository(db *gorm.DB) BankAccountRepository {
...
 	err := p.DB.Find(&bankAccounts).Error
 	return bankAccounts, err
 }
 
 // FindAll ...
 func (p *BankAccountRepository) FindAll(argsStr map[string]string, argsInt map[string]int) ([]model.BankAccount, error) {
-	bankAccounts := []model.BankAccount{}
+	var bankAccounts []model.BankAccount
```
Comment (yakuter) [inline internal/storage/bank_account_repository.go:29]: Same issue.

File: internal/storage/bank_account_repository.go [hunk 3]
```diff
@@ -52,7 +52,7 @@ func (p *BankAccountRepository) FindAll(argsStr map[string]string, argsInt map[s
 
 // FindByID ...
 func (p *BankAccountRepository) FindByID(id uint) (model.BankAccount, error) {
-	bankAccount := model.BankAccount{}
+	var bankAccount model.BankAccount
```
Comment (yakuter) [inline internal/storage/bank_account_repository.go:55]: Same issue.

File: internal/storage/credit_card_repository.go [hunk 4]
```diff
@@ -19,14 +19,14 @@ func NewCreditCardRepository(db *gorm.DB) CreditCardRepository {
 
 // All ...
 func (p *CreditCardRepository) All() ([]model.CreditCard, error) {
-	creditCards := []model.CreditCard{}
+	var creditCards []model.CreditCard
```
Comment (yakuter) [inline internal/storage/credit_card_repository.go:22]: Same issue.

File: internal/storage/credit_card_repository.go [hunk 5]
```diff
@@ -19,14 +19,14 @@ func NewCreditCardRepository(db *gorm.DB) CreditCardRepository {
...
 	err := p.DB.Find(&creditCards).Error
 	return creditCards, err
 }
 
 // FindAll ...
 func (p *CreditCardRepository) FindAll(argsStr map[string]string, argsInt map[string]int) ([]model.CreditCard, error) {
-	creditCards := []model.CreditCard{}
+	var creditCards []model.CreditCard
```
Comment (yakuter) [inline internal/storage/credit_card_repository.go:29]: Same issue.

**Comment by yakuter** [comment]
Hello @mrturkmen06 these are nice changes and I would like to see you on board. There is just a little thing I want you to revert which I reviewed. Then I will merge this nice PR. Thank you for great work.

File: internal/storage/bank_account_repository.go (hunk 3 above)
Comment (mrturkmen06) [reply internal/storage/bank_account_repository.go:55]: If the record is not exist on db, then response for endpoint `http://localhost:3625/api/bank-accounts/1` will be 
```
{
    "code": 404,
    "status": "Error",
    "message": "record not found",
    "errors": null
}
```
is not ok ?

File: internal/storage/credit_card_repository.go (hunk 4 above)
Comment (mrturkmen06) [reply internal/storage/credit_card_repository.go:22]: Same, it does not cause to nill.

File: internal/storage/credit_card_repository.go (hunk 5 above)
Comment (mrturkmen06) [reply internal/storage/credit_card_repository.go:29]: If the record is not exist on db, then response will be
```
{
    "code": 404,
    "status": "Error",
    "message": "record not found",
    "errors": null
}
```
is not ok ?

File: internal/storage/bank_account_repository.go (hunk 1 above)
Comment (mrturkmen06) [reply internal/storage/bank_account_repository.go:22]: I guess, I have tested it and it was working pretty ok. I did not manage to get null , from any of them. I have tested following endpoint without  having any bank account.

`http://localhost:3625/api/bank-accounts`

if it is possible, could you give me PoC example ? for that scenario which I can understand the problem better. When you have time, of course.

File: internal/storage/bank_account_repository.go (hunk 3 above)
Comment (yakuter) [reply internal/storage/bank_account_repository.go:55]: You are right about this response.

**Comment by mrturkmen06** [comment]
Hi @yakuter, Thank you for this nice project and reviewing the changes. I would like to get more information about development process about the project, like, dev is in back than master branch, usually, it is vice versa. I have no idea about development cycle  that could be nice to mention about it in somewhere. 

You caught  me while I was writing ... :)

**Pull request merged**